### PR TITLE
Clarify permissions required to approve PRs

### LINKED ISSUE #20989: Document permission required to `approve` ✅ a pull request
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role

### What part(s) of the article would you like to see updated?

[`Permissions for each role`](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role) lists many roles, including:

* Merge pull requests on protected branches, even if there are no approving reviews

Oddly, it doesn't document `approving` itself.

### Additional information

We have a repository where we wanted to give someone `triage` permission, it seemed like it should be enough for them to approve a PR w/o giving them write access.

[Repository roles for organizations](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#repository-roles-for-organizations) lists:

> Triage: Recommended for contributors who need to proactively manage issues and pull requests without write access

`Triage` _sounds_ like the right permission. And the general overview fits. But...

In testing, it appears that `triage` is insufficient to grant meaningful approval as shown here:
image

Given that a repository can easily choose who has write access and who has triage, and who has neither, it doesn't seem unreasonable to expect that someone who could triage could also approve, w/o themselves being able to push directly to a repository.

That said. It is unreasonable to not be able to figure out from the documentation which role is necessary to actually approve.

### PR DESCRIPTION
**Pull request by jsoref**
### Why:

Closes #20989

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Screenshot_20221019-142703

Collecting all PR review items into a single place to make it easier for people to understand.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @jsoref Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by isaacmbrown — DISMISSED** [review]
@jsoref Thanks very much for your speedy work to open the PR to address the issue! Just going to apply a couple of changes and we should be good to go 🚀

File: content/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization.md [hunk 1]
```diff
@@ -73,7 +73,9 @@ Some of the features listed below are limited to organizations using {% data var
 | Reopen issues they closed themselves | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
 | Have an issue assigned to them | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
 | Send pull requests from forks of the team's assigned repositories | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
-| Submit reviews on pull requests | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
+| [Submit reviews on pull requests](/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
+| [Submit reviews that affect a pull request's mergeability](/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews) | | | **✔️** | **✔️** | **✔️** |
```
Comment (jsoref) [inline content/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization.md:77]: Not having the phrases `approve` or `require changes` means that searching for the things normal users experience doesn't work...

Comment (isaacmbrown) [reply content/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization.md:77]: @jsoref That's a fair point. How about `Approve or request changes on a pull request with required reviews`?

Comment (jsoref) [reply content/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization.md:77]: I'll take that.

**Review by isaacmbrown — APPROVED** [review]
Looking good on staging 🚀

**Review by nea006 — COMMENTED** [review]
ไปให้ไกล้ๆ

**Comment by robledo6** [comment]
Ok

**Pull request merged**